tools/ci-analyzer.py

```
@@ -0,0 +1,243 @@
+    return token
+
+
+def api_get(endpoint, token):
+    url = f"{GITHUB_API}{endpoint}"
+    req = urllib.request.Request(url, headers={
+        "Authorization": f"token {token}",
+        "Accept": "application/vnd.github.v3+json",
+        "User-Agent": "llvm-ci-analyzer",
+    })
+    try:
+        with urllib.request.urlopen(req) as resp:
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)